rachelnavarro/step/portfolio/src/main/webapp/index.html

```diff
@@ -20,62 +20,63 @@
       </div>
     </div>
     <div id="content">
-      <h1>Rachel Navarro's Portfolio</h1>
+      <h1><span class="typewrite">Rachel Navarro's Portfolio</span></h1>
```

Comment:
why is all this stuff here?

Reply:
I added some style features to the web portfolio in-between this pr and the last pr, so I just decided to include those changes/commits in this pr. But in the future, I can just make these changes belong to their own pr just to make it less confusing!